rviz_default_plugins/package.xml

```diff
@@ -35,6 +35,7 @@
   <test_depend>ament_cmake_gmock</test_depend>
   <test_depend>ament_cmake_lint_cmake</test_depend>
   <test_depend>ament_cmake_uncrustify</test_depend>
+  <test_depend>rviz_rendering_tests</test_depend>
```

Comment:
You'll need a `<test_depend>` or `<depend>` (if you use it in the source and not just the tests) on `visualization_msgs`.

Reply:
Clearly, yes. Thanks for noting that oversight on my part!